### ISSUE TITLE
Incompatible with 24.2

### ISSUE THREAD

**Issue by orschiro** [gnome-integration-team/thunderbird-gnome#13]
Is this theme still maintained?

I tried to install it but it is incompatible with Thunderbird 24.2.0.

Thanks!

**Comment by pprw** [comment]
+1 (same problem)

I believe all debian unstable users have this issue.

**Comment by orschiro** [comment]
@pprw 

I am not using Debian but Arch Linux and I doubt it is related to any distribution but rather Thunderbird 24.2.

**Comment by pprw** [comment]
@orschiro Of course. My point was : every user of debian unstable should experiment this problem today because the versions of thunderbird (icedove) in the unstable repository is the one (24.2)  which causes the issue.

**Comment by pprw** [comment]
I just tried to modify the extension and change the maximum thunderbird version (it was 9.0 and I put 24.2). It allows to install the extension but the result is not beautiful.

This extension needs an update.

**Comment by orschiro** [comment]
@pprw 

Can you post a screenshot? Then maybe we can find out how to fix the glitches.

**Comment by antistress** [comment]
Here is a screenshot of Icedove on Debian Sid+Experimental with the theme
![thunderbird24](https://f.cloud.github.com/assets/2436957/2047287/23e7a274-8a19-11e3-900f-9b2d9ef1c92a.png)

**Comment by jtts** [comment]
Pull request #12 provides support for Thunderbird 24. After a quick test it seems to work OK. The pull request is six months old. Does anyone know why it hasn’t been merged yet?

**Comment by orschiro** [comment]
Great to see it being merged!

Can someone also please update the extension page?

https://addons.mozilla.org/en-US/thunderbird/addon/adwaita-thunderbird/
